WalletWasabi/Extensions/LinqExtensions.cs

```diff
@@ -67,15 +67,17 @@ public static void Shuffle<T>(this IList<T> list)
 			}
 		}
 
-		public static void AddToValueList<TKey, TValue, TElem>(this Dictionary<TKey, TValue> myDic, TKey key, TElem elem) where TValue : List<TElem>
+		public static void AddToValueList<TKey, TValue, TElem>(this Dictionary<TKey, TValue> myDic, TKey key, TElem elem)
+			where TKey: notnull
+			where TValue : List<TElem>
```

Comment:
Wouldn't be possible to change this to something like the following in order to remove the castings?
```suggestion
		public static void AddToValueList<TKey, TElem>(this Dictionary<TKey, List<TElem>> myDic, TKey key, TElem elem)
			where TKey: notnull
```

Reply:
Yes, good point. Updated.